src/loaders/genome_attributes/parse_computed_genome_attribs.py

```diff
@@ -0,0 +1,247 @@
+"""
+PROTOTYPE
+
+This script involves processing computed genome attributes and organizing them into a structured format suitable
+for importing into ArangoDB. The resulting JSON file will be used to update (upsert/insert) the database with the parsed
+genome attribute data.
+
+Note: If the ArangoDB collection has been previously created using a JSON file generated by the genome attributes
+      loader script, it is crucial to ensure that the arguments "--load_ver" and "--kbase_collection" are consistent
+      with the ones used in the genome attributes loader script in order to ensure that the same key is generated
+      for the corresponding Arango document."
+
+usage: parse_computed_genome_attribs.py [-h] --load_ver LOAD_VER --kbase_collection KBASE_COLLECTION
+                                        [--tools TOOLS [TOOLS ...]] [--root_dir ROOT_DIR] [-o OUTPUT]
+
+options:
+  -h, --help            show this help message and exit
+
+required named arguments:
+  --load_ver LOAD_VER   KBase load version (e.g. r207.kbase.1).
+  --kbase_collection KBASE_COLLECTION
+                        KBase collection identifier name.
+
+optional arguments:
+  --tools TOOLS [TOOLS ...]
+                        Extract results from tools. (default: retrieve all available sub-directories in the [load_ver]
```

Comment:
At some point we need to document the folder structure in a *.md file in the repo, along with the process for adding a collection and uploading to Arango

Reply:
👍 listed in ticket https://kbase-jira.atlassian.net/browse/RE2022-81

Reply:
:+1: